crates/anvil-polkadot/tests/it/time_machine.rs

```diff
@@ -113,6 +147,30 @@ async fn test_evm_increase_time() {
         1,
         "Wrong timestamp",
     );
+    assert_with_tolerance(
+        unwrap_response::<i64>(
+            node.eth_rpc(EthRequest::EvmIncreaseTime(U256::from(3600))).await.unwrap(),
+        )
+        .unwrap(),
+        7200,
+        1,
+        "Wrong offset when increasing the timestamp.",
+    );
+    let _ = node.eth_rpc(EthRequest::Mine(None, None)).await.unwrap();
+    let third_hash = node.block_hash_by_number(3).await.unwrap();
+    let third_timestamp = node.get_decoded_timestamp(Some(third_hash)).await;
+    assert_with_tolerance(
+        third_timestamp.saturating_sub(first_timestamp).saturating_div(1000),
+        7200,
+        1,
```

Comment:
nit: this tolerance and the one below seem a bit too little. Test my end up being quite flaky. Haven't you observed?